src/main/views/home.njk

```diff
@@ -1,5 +1,106 @@
 {% extends "template.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/file-upload/macro.njk" import govukFileUpload %}
+{% from "govuk/components/select/macro.njk" import govukSelect %}
+{% from "govuk/components/textarea/macro.njk" import govukTextarea %}
+
 
 {% block content %}
-  <h1 class="govuk-heading-xl">Default page template</h1>
+  <h1 class="govuk-heading-xl">Bulk scan upload</h1>
+
+  <form action="/" method="POST" enctype="multipart/form-data">
+
+    {{ govukSelect({
+      id: "environment",
+      name: "environment",
+      label: {
+        text: "Please select environment:"
+      },
+      items: [
+        {
+          value: "aat",
+          text: "aat"
+        },
+        {
+          value: "demo",
```

Comment:
still needs a local option :)

Reply:
added local option

Reply:
local upload will be part of new ticket for now i think